client/blocks/app-promo/index.jsx

```diff
@@ -110,16 +110,9 @@ export class AppPromo extends React.Component {
 
 	sendMagicLink = () => {
 		this.recordClickEvent();
-
 		const email = this.props.userSettings.user_email;
-		wpcom.undocumented().requestMagicLoginEmail( {
-			email,
-			infer: true,
-			scheme: 'wordpress',
-		} );
-
+		this.props.sendEmailLogin( email, { showGlobalNotices: false, isMobileAppLogin: true } );
```

Comment:
Shouldn't we defer `isMobileAppLogin` to `this.state.promoItem.type === 'mobile'`? Or am I'm missing the intent of the `promoItem` object?

Reply:
Nevermind, just noted `sendMagicLink` is only called from `mobilePromo`